lua/starfall/sflib.lua

```diff
@@ -102,6 +102,8 @@ function SF.throw ( msg, level, uncatchable )
 	error( err )
 end
 
+local real_type = function () end
```

Comment:
Don't know if it would make more sense, maybe just the same amount of sense haha. It would perform in the same way so I don't really think its an issue but I can change it if you would like?